front/app/containers/ProjectsShowPage/timeline/VotingResults/index.tsx

```diff
@@ -57,14 +57,7 @@ const VotingResults = ({ phaseId, votingMethod }: Props) => {
           <VotingResultCard
             idea={idea}
             phaseId={phaseId}
-            rank={
-              getRanks(
-                getCounts(
-                  ideasList,
-                  votingMethod === 'budgeting' ? 'baskets_count' : 'votes_count'
-                )
-              )[i]
-            }
+            rank={getRanks(getCounts(ideasList, 'total_votes'))[i]}
```

Comment:
So for participatory budgeting, `'total_votes'` and the manual votes actually represent baskets?

Reply:
Also maybe it would be cleaner to run this outside of the loop, since it only needs to be generated once. So e.g.

```
const ranks = getRanks(getCounts(...));

...

rank={ranks[i]}

Reply:
Indeed - for PB, total_votes = total_picks (I.g. Total # baskets which have this idea in it).

For other voting methods, total_votes = total # votes for this idea.